lib/formatters/index.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2012 Mark Cavage, Inc.  All rights reserved.
```

Comment:
What was the reason behind removing the copyright? Did your IDE remove these automatically?